Cktgen/runit-comparator.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
 
-(cd Intel; python generate_comparator_v2.py)
+(cd Intel; python3 generate_comparator_v2.py)
```

Comment:
I always run in a virtual enviornment so I don't need to do this.

Reply:
How about we add a condition to the script then such as below?
```
PYVER=`python --version`
PYVER_REQ='Python 3'
if [ "$PYVER" \< "$PYVER_REQ" ];
then
	echo "$PYVER_REQ or above is required"
	exit 1
fi
```